.circleci/config.yml

```diff
@@ -0,0 +1,103 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/project/yarn
+  docker:
+    - image: yarnpkg/dev:latest
+
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          key: yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install Dependencies
+          command: yarn install
+      - save_cache:
+          key: yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}
```

Comment:
- The `{{ .Branch }}` specifier here seems useless as the checksum will always be more specific than the branch name.
- Also, it is always interesting to have fallbacks if the specific cache key is not found (e.g.: `yarn-`, `yarn-{{ .Branch }}-` and `yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}`)

Reply:
Fixed! 6ec592a